lib/checks/navigation/region.js

```diff
@@ -62,7 +74,26 @@ function findRegionlessElms(virtualNode) {
 	}
 }
 
-var regionlessNodes = findRegionlessElms(virtualNode);
-this.relatedNodes(regionlessNodes);
+regionlessNodes = findRegionlessElms(virtualNode)
+	// Find first parent marked as having region descendant (or body) and
+	// return the node right before it as the "outer" element
+	.map(node => {
+		let vNode = axe.utils.getNodeFromTree(node);
+		while (
+			vNode.parent &&
+			!vNode.parent._hasRegionDescendant &&
+			vNode.parent.actualNode !== document.body
+		) {
+			vNode = vNode.parent;
+		}
+
+		return vNode.actualNode;
+	})
+	// Remove duplicate containers
+	.filter((node, index, array) => {
+		return array.indexOf(node) === index;
+	});
+this.data(regionlessNodes);
```

Comment:
Why are you putting these into data? Other than for testing, I don't see what this is useful for. And since this will end up in our output data, I don't think we should use it this way.